intro/setup/index.html

```diff
@@ -0,0 +1,349 @@
+    <p>
+        As pygame is a library for <i>python</i>, it is necessary that you 
+        are familiar with Python itself. This goes beyond the scope of this book, but 
+        there are some other great resources for learning Python, such  
+        as <a href="">automate the boring stuff</a>.
+        
+        Now, luckily you don't have to be an Anthony Shaw or Tim Peters to be
+        able to use pygame, just some basic Python will do. 
+        This includes:
+            <ul>
+                <li>Understanding how types and instances work</li>
+                <li>Knowing how to access and modify attributes of objects</li>
+                <li>Familiarity with basic OOP(Object Oriented Programming)</li>
+                <li>Basic control flow keywords such as - for, while, break, if, elif, else</li>
+                <li>Functions</li>
+            </ul>
+        
+        Since in Python everything is an object, that is also how you would 
+        interact with the utilities provided through pygame, so knowing how they work 
+        is crucial.
+    </p>
+
+    <h2><a id="installation" class="js"></a>Installing pygame</h2>
+    <p>
+        pygame can be installed through <a href="https://pypi.org">pypi</a> by
+        using the pip package manager.
+
+        This is as simple as,
+    </p>
+
+    <p>
+{% highlight python %}
+pip install pygame
+{% endhighlight %}
+
+    BUT, how you access your pip depends on your Operating System and what interpreter 
+    you plan on using to run the code you will be writing throughout this article. 
+
+    For the sake of this article, we recommend installing pygame globally, and using your global interpreter 
+    to run your code. 
+    
+    For windows, the way you access pip can be 
+
+
+{% highlight python %}
+pip install pygame 
+# Or
+
+# Accessing pip as a module through your python launcher
+py -m pip install pygame 
+
+python -m pip install pygame
+{% endhighlight %}
+
+    If you aren't too confident with where pygame is being installed, use your python launcher 
+    as shown in the last 2 methods to install it, and use the same launcher to run the code accordingly!
+
+    </p>
+
+    <h2><a id=""></a></h2>
+
+    <h2><a id="project-directory" class="js"></a>Setting up the project directory</h2>
+    <p>
+        Throughout this whole miniseries, you will be required to work with a couple different 
+        files, it would be better and recommended if everything required to run your game is within 
+        one directory. We will create the required folders as we go, rather than take the generic 
+        approach and dump everything into our project folder that we wouldn't need(yet). But, 
+        we do still need some basic setup.
+    </p>
+
+    <p>
+        First, create a folder whereever you would like, and name it whatever you want. I will be
+        naming it <b>pygame_platformer</b> and I will store it in my <b>d:\dev\projects\</b> directory.
```

Comment:
Capitalizing `D:` is probably a good idea because that's what Windows does and people relatively new to working with file paths and projects may not realize that drive letters are not case-sensitive